docs/source/llms/langchain/index.rst

```diff
@@ -240,3 +240,89 @@ How can I use a streaming API with LangChain?
     As of the MLflow 2.12.2 release, LangChain models that support streaming responses that have been saved using MLflow 2.12.2 (or higher) can be loaded and used for 
     streamable inference using the ``predict_stream`` API. Ensure that you are consuming the return type correctly, as the return from these models is a ``Generator`` object.
     To learn more, refer to the `predict_stream guide <https://mlflow.org/docs/latest/models.html#how-to-load-and-score-python-function-models>`_.
+
+How can I log my chain from code?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Models from Code**: MLflow 2.12.2 introduced the ability to log LangChain models directly from a code definition. 
```

Comment:
```suggestion
- Models from Code: MLflow 2.12.2 introduced the ability to log LangChain models directly from a code definition. For more details about this feature, please refer to `Models from Code <../../models#models-from-code>`_ documentation.
```
nit: so we don't need to duplicate additional contents (e.g. thread-safe warnings) between two pages.